db/migrate/20150610155632_add_image_to_guest_application.rb

```diff
@@ -0,0 +1,9 @@
+class AddImageToGuestApplication < ActiveRecord::Migration
+  def up
```

Comment:
@dkorn please consolidate this to just:

    def change
      add_column :guest_applications, :container_image_id, :bigint
    end